### ISSUE TITLE
[Feature Request] Spinners for the Remote

### ISSUE THREAD

**Issue by ThizThizzyDizzy** [TeamPneumatic/pnc-repressurized#360]
Spinners would allow players to input specific coordinates into a variable. This could be used to specify delivery locations, how many items should be transported, and more
Here's a mock-up of what it might look like:
![image](https://user-images.githubusercontent.com/22590237/60741312-eb862c00-9f36-11e9-8c93-ed9074b71828.png)

**Comment by desht** [comment]
Good idea, but would really require a new widget type to be created.  This may need to wait till 1.14.x.
